dts/arm/nxp/nxp_rt6xx.dtsi

```diff
@@ -18,6 +15,9 @@
 		peripheral: peripheral@50000000 {
 			ranges = <0x0 0x50000000 0x10000000>;
 		};
+		flexspi: spi@134000 {
```

Comment:
Maybe `flexspi: spi@5134000`?

Reply:
This will cause problems with how we are referencing this node in Kconfig.defconfig for the FLASH_SIZE.